external/java/src/main/java/ch/epfl/dedis/byzcoin/ByzCoinRPC.java

```diff
@@ -83,18 +91,22 @@ public ByzCoinRPC(Roster r, Darc d, Duration blockInterval) throws CothorityExce
      * @param skipchainId the ID of the genesis skipblock, aka skipchain ID
      * @throws CothorityException
      */
-    public ByzCoinRPC(Roster roster, SkipblockId skipchainId) throws CothorityException, InvalidProtocolBufferException {
+    public ByzCoinRPC(Roster roster, SkipblockId skipchainId) throws CothorityException{
```

Comment:
In the instances, we have `from*` when connecting to an existing instances, the constructor is only used for creating new instances. Do we want something similar for `ByzCoinRPC` too? This one is connecting to an existing skipchain ID, maybe use `fromExisting(Roster roster, SkipblockId skipchainId)`?